src/duration.rs

```diff
@@ -232,6 +232,20 @@ impl Duration {
         }
     }
 
+    /// Returns the number of nanoseconds such that
+    /// `subsec_micros() + num_seconds() * MICROS_PER_SEC` is the total number of
+    /// microseconds in the duration.
+    pub const fn subsec_micros(&self) -> i32 {
+        self.subsec_nanos() / NANOS_PER_MICRO
+    }
+
+    /// Returns the number of nanoseconds such that
```

Comment:
> Returns the number of *nano*seconds such that ...

I suspect this docstring and the docstring above are copy+pasta from `subsec_nanos`. 🤨

Reply:
Oof, yeah. I missed that part when I was reading through. Thanks for catching that